### PR TITLE
Pa-2409 Remove Zebra Feeding Station from list of predominate uses

### PR DESCRIPTION
**Pull request by agalbrai**
I think I did this right?

### CODE REVIEW

**Review by Fosol — APPROVED** [review]
You will need to make sure the date the file was created is after the one Jonny created `v01.05.04` so that they are installed in the correct order.  Additionally you will need to merge this update after Jonny's is merged.

**Review by jdtoombs — APPROVED** [review]

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**
